package/MDAnalysis/coordinates/base.py

```diff
@@ -216,6 +224,9 @@ def __init__(self, n_atoms, **kwargs):
         self.has_forces = kwargs.get('forces', False)
 
         self._unitcell = self._init_unitcell()
+        
+        # set up aux namespace for adding auxiliary data
+        self.aux = Namespace()
```

Comment:
It depends on how we want to access the auxiliary data. In all the discussions we assumed something like `ts.aux.force`; this requires a class as what @fiona-naughton used. `ts.aux['force'] `is fine by me.